### PR TITLE
Update emeritus following the community guideline

### PR DESCRIPTION
**Pull request by reyang**
Discussed during the Maintainers SIG meeting. Applying https://github.com/open-telemetry/community/blob/main/community-membership.md#emeritus-maintainerapprovertriager.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -89,38 +89,29 @@ doc](https://docs.google.com/document/d/1yjjD6aBcLxlRazYrawukDgrhZMObwHARJbB9glW
...
+* [Robert Pajak](https://github.com/pellared), Splunk
+* [Utkarsh Umesan Pillai](https://github.com/utpilla), Microsoft
 
-* [Victor Lu](https://github.com/victlu), Microsoft
+[Emeritus
+Maintainer/Approver/Triager](https://github.com/open-telemetry/community/blob/main/community-membership.md#emeritus-maintainerapprovertriager):
 
-*Find more about the triager role in [community
-repository](https://github.com/open-telemetry/community/blob/main/community-membership.md#triager).*
+* [Bruno Garcia](https://github.com/bruno-garcia)
```
Comment (cijothomas) [inline README.md:109]: https://github.com/open-telemetry/opentelemetry-dotnet/pull/1312

**Comment by cijothomas** [comment]
@bruno-garcia @eddynaka @MikeGoldsmith @pjanotti @SergeyKanzhelev @victlu @lmolkova Please take a look and request changes, if any.

**Review by eddynaka — APPROVED** [review]

**Comment by eddynaka** [comment]
> @bruno-garcia @eddynaka @MikeGoldsmith @pjanotti @SergeyKanzhelev @victlu @lmolkova Please take a look and request changes, if any.

Sounds good!
thanks! :)

**Review by SergeyKanzhelev — APPROVED** [review]
thank you!

**Review by bruno-garcia — APPROVED** [review]
Thanks!

**Comment by MikeGoldsmith** [comment]
Thanks, I agree with this change too 👍🏻

**Review by MikeGoldsmith — APPROVED** [review]

**Review by pjanotti — APPROVED** [review]
LGTM. Thanks for the pings.

**Review by cijothomas — APPROVED** [review]

**Pull request merged**